File: Pset6/mario/less/mario.py
# import the function get_int from cs50 module
from cs50 import get_int

# prompt and validate user input
while True:
    height = get_int("Height: ")

    # chech condition
    if height > 0 and height <= 8 and type(height) == int:
        break

# row number
row = 1

# print the pyramid
for i in range(height):
    print(" "*(height-row), end="")
    print("#"*row, end="")
    row += 1
    print()

File: Pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;    // height of our pyramid
    int hash = 1 ;  // number of hash

    // get the height from the user
    do
    {
        height = get_int("Height :");
    }
    while (height <= 0 || height > 8);

    while (height >= 1)
    {
        // print spaces
        int i, j;
        for (i = 0; i < height - 1; i++)
        {
            printf(" ");
        }

        // print hashes
        for (j = 0; j < hash; j++)
        {
            printf("#");
        }
      
        printf("\n");
        hash++;
        height--;
    }
}
